arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/CreateSection.java

```diff
@@ -132,6 +134,7 @@ public CreateSection(Hub hub) {
     // OOGX case
     if (Exceptions.outOfGasException(exceptions)) {
       scenarioFragment.setScenario(CREATE_EXCEPTION);
+      commonValues.setTracedException(TracedException.OUT_OF_GAS_EXCEPTION);
```

Comment:
Like for `CALL`'s I think nothing is stopping us from including the `CREATE` family in the high-level `TracedException` setting.